File: WEEK2/bj2_1427.py
import sys

lines = sys.stdin.readline().strip()
arr = [int(char) for char in lines]

for i in range(len(arr)):
    max_idx = i
    for j in range(i + 1, len(arr)):
        if arr[j] > arr[max_idx]:
            max_idx = j
    arr[i], arr[max_idx] = arr[max_idx], arr[i]

for i in arr:
    print(i, end="")

File: WEEK2/bj2_1427.js
const fs = require('fs');
const input = fs.readFileSync('/dev/stdin').toString().trim();

const arr = input.split('').map(Number);

for (let i = 0; i < arr.length; i++) {
    let maxIdx = i;
    for (let j = i + 1; j < arr.length; j++) {
        if (arr[j] > arr[maxIdx]) {
            maxIdx = j;
        }
    }
    let temp = arr[i];
    arr[i] = arr[maxIdx];
    arr[maxIdx] = temp;
}

console.log(arr.join(''));
